src/js/rx/reducers/disclaimer.js

```diff
@@ -0,0 +1,18 @@
+import set from 'lodash/fp/set';
+
+const initialState = {
+  visible: true
+};
+
+export default function disclaimer(state = initialState, action) {
+  switch (action.type) {
+    case 'OPEN_DISCLAIMER':
+      return Object.assign({}, state, {
```

Comment:
Just `set('visible', true, state)` would be fine. Actually, I think I'm going to change the `Object.assign` I used at some point to `merge` from `lodash/fp/merge`.